docs/hub/sample-factory.md

```diff
@@ -0,0 +1,96 @@
+# Using sample-factory at Hugging Face
+
+`sample-factory` is a codebase for high throughput asynchronous reinforcement learning. It has integrations with HuggingFace Hub to push models with evaluation results and training metrics to the hub.
+
+
+## Exploring sample-factory in the Hub
+
+You can find `sample-factory` models by filtering at the left of the [models page](https://huggingface.co/models?library=sample-factory).
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
+2. Metadata tags that help for discoverability.
+3. Evaluation results to compare with other models.
+4. A video widget where you can watch your agent performing.
+
+
+## Install the library
+To install the `sample-factory` library, you need to install the package:
+
+`pip install sample-factory
+
+SF is known to work on Linux and macOS. There is no Windows support at this time.`
```

Comment:
Closing backtick in the wrong spot:

`pip install sample-factory`

SF is known to work on Linux and macOS. There is no Windows support at this time.

Reply:
Other than that, everything else looks good.